FreeAPS/Sources/Localizations/Main/en.lproj/Localizable.strings

```diff
@@ -2010,7 +2010,7 @@ Enact a temp Basal or a temp target */
 "Insulin Peak Time" = "Insulin Peak Time";
 
 /* "Insulin Peak Time" */
-"Time of maximum blood glucose lowering effect of insulin, in minutes. Beware: Oref assumes for ultra-rapid (Lyumjev) & rapid-acting (Fiasp) curves minimal (35 & 50 min) and maximal (100 & 120 min) applicable insulinPeakTimes. Using a custom insulinPeakTime outside these bounds will result in issues with Trio, longer loop calculations and possible red loops." = "Time of maximum blood glucose lowering effect of insulin, in minutes. Beware: Oref assumes for ultra-rapid (Lyumjev) & rapid-acting (Fiasp) curves minimal (35 & 50 min) and maximal (100 & 120 min) applicable insulinPeakTimes. Using a custom insulinPeakTime outside these bounds will result in issues with Trio, longer loop calculations and possible red loops.";
+"Requires 'Use Custom Peak Time' to be enabled. The time, in minutes, of maximum blood glucose lowering effects of insulin. Beware: Oref assumes for ultra-rapid (Lyumjev) & rapid-acting (Fiasp) curves minimal (35 & 50 min) and maximal (100 & 120 min) applicable insulinPeakTimes. Using a custom insulinPeakTime outside these bounds will result in issue, longer loop calculations and possible red loops." = "Requires 'Use Custom Peak Time' to be enabled. The time, in minutes, of maximum blood glucose lowering effects of insulin. Beware: Oref assumes for ultra-rapid (Lyumjev) & rapid-acting (Fiasp) curves minimal (35 & 50 min) and maximal (100 & 120 min) applicable insulinPeakTimes. Using a custom insulinPeakTime outside these bounds will result in issue, longer loop calculations and possible red loops.";
```

Comment:
There always (previously?) a bug where the app would hang if entering peak times outside of the mentioned limits. I think there are guardrails in place for this now, but please check if that’s the case. If so, the “Beware …” text can be removed. The guardrails are likely applied without notifying the user, which could be improved.